src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs

```diff
@@ -514,7 +514,7 @@ public override void ExecuteCmdlet()
 
                 // retrieve encryption state using per-disk instance status and report if successful
                 AzureDiskEncryptionStatusContext encryptionStatus = GetStatusFromInstanceView();
-                if (encryptionStatus != null && !(encryptionStatus.OsVolumeEncrypted==EncryptionStatus.Unknown || encryptionStatus.DataVolumesEncrypted==EncryptionStatus.Unknown))
+                if (encryptionStatus != null && (encryptionStatus.OsVolumeEncrypted!=EncryptionStatus.Unknown || encryptionStatus.DataVolumesEncrypted!=EncryptionStatus.Unknown))
                 {
                     encryptionStatus.ProgressMessage = progressMessage;
                     WriteObject(encryptionStatus);
```

Comment:
Please revert changes to the psm1 file below.

Reply:
reverted

Reply:
@maddieclayton all checks have passed with the reverted file